src/style.css

```diff
@@ -0,0 +1,56 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.search-big,
+.search-small {
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.input-big,
+.input-small {
+  background: url(images/Search.svg) 25px center/19px 19px no-repeat;
+  padding-left: 62px;
+  width: 100%;
+  height: 68px;
+  font-size: 16px;
+  font-weight: 300;
+  border: none;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
+}
+
+.input-small {
+  height: 40px;
+  font-size: 14px;
+  padding-left: 33px;
+  background-size: 11px 11px;
+  background-position: 13px center;
+}
+
+.input-big::placeholder,
+.input-small::placeholder {
+  color: #3d4e61;
+}
+
+.search-big:hover,
+.search-small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input-big:focus,
+.input-small:focus {
+  background: url(images/Search.svg) 25px center/19px 19px no-repeat, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
change only background position

Reply:
do i need to make an icon as "img"? because I made it as a background and because of this I need to duplicate all the code in :focus, and without it the image changes to a linear-gradient